### ISSUE TITLE
Stamina Addition

### ISSUE THREAD

**Issue by TJDNOW** [littlebudddy321/New-Lands-RPG#2]
Found project on reddit and would like to contribute a stamina bar or maybe a day / night cycle so the stores arent open 24/7

**Comment by littlebudddy321** [comment]
XD, that’s a good suggestion, did you mean YOU were going to contribute by making it or did you just contribute the idea?

**Comment by TJDNOW** [comment]
Yeah id like to contribute but i have no clue how to use Github lmao
